### DISCUSSION TITLE
During the model_devi process, an error occurred that lost atoms

### DISCUSSION THREAD

**Discussion by manbujin** [deepmodeling/dpgen#1071] [Q&A]
The problem of missing atoms arose during lammps calculations.
This problem may be due to insufficient training, I want to continue running for the next iteration, how can I solve this problem？
Thank you!

**Comment by HuangJiameng** [comment]
You can relax the dataset.

**Reply by manbujin** [reply]
Thank you for your reply, but can you be more specific, I don't quite understand what you mean.
It terminated the calculation at this step of exploration, and the reason shown was the loss of atoms, I want to continue the calculation, what should I do.

**Reply by HuangJiameng** [reply]
In which iteration this problem occurs?

**Comment by hyuqing11** [comment]
I encountered a similar issue when using DPGEN for active learning. During the model_devi tasks in LAMMPS, I received the error "ERROR: Lost atoms". Do you have any suggestions on how I can continue the calculation for exploration?

**Reply by manbujin** [reply]
I turned the step size down to 0.0005 and it doesn't have such an error again. Or manually delete the failed example and let it continue to calculate.

**Reply by hyuqing11** [reply]
I see. Thank you for your help!
